tagmanager/src/tm_tag.h

```diff
@@ -80,7 +80,10 @@ typedef enum
 #define TAG_IMPL_VIRTUAL 'v' /**< Virtual implementation */
 #define TAG_IMPL_UNKNOWN 'x' /**< Unknown implementation */
 
-typedef struct _TMTag
+/**
+ * The TMTag structure represents a single tag in the tag manager.
+ **/
+typedef struct TMTag
```

Comment:
Uhm, for consistency with other TM* structs. Not sure if it's truly necessary for gir (probably not)